naming/src/test/java/com/alibaba/nacos/naming/healthcheck/v2/processor/HealthCheckProcessorV2DelegateTest.java

```
@@ -0,0 +1,98 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class HealthCheckProcessorV2DelegateTest {
+    
+    @Mock
+    private HealthCheckExtendProvider healthCheckExtendProvider;
+    
+    @Mock
+    private HealthCheckTaskV2 healthCheckTaskV2;
+    
+    @Mock
+    private Service service;
+    
+    @Mock
+    private ClusterMetadata clusterMetadata;
+    
+    private HealthCheckProcessorV2Delegate healthCheckProcessorV2Delegate;
+    
+    @Before
+    public void initBean() {
+        healthCheckProcessorV2Delegate = new HealthCheckProcessorV2Delegate(healthCheckExtendProvider);
+        verify(healthCheckExtendProvider).init();
+    }
+    
+    @Test
+    public void testAddProcessor() {
+        List<HealthCheckProcessorV2> list = new ArrayList<>();
+        list.add(new TcpHealthCheckProcessor(null, null));
+        healthCheckProcessorV2Delegate.addProcessor(list);
+        
+        Class<HealthCheckProcessorV2Delegate> healthCheckProcessorV2DelegateClass = HealthCheckProcessorV2Delegate.class;
+        try {
+            Field field = healthCheckProcessorV2DelegateClass.getDeclaredField("healthCheckProcessorMap");
+            field.setAccessible(true);
+            Map<String, HealthCheckProcessorV2> map = (Map<String, HealthCheckProcessorV2>) field
+                    .get(healthCheckProcessorV2Delegate);
+            HealthCheckProcessorV2 healthCheckProcessorV2 = map.get(HealthCheckType.TCP.name());
+            Assert.assertNotNull(healthCheckProcessorV2);
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            e.printStackTrace();
```

Why ignore exception? if throw exception is expected behavior, should add it in @Test(expected=xxx)